schema/parse.go

```diff
@@ -433,8 +456,12 @@ func isTypeDeclaration(item lex.Item, it *lex.ItemIterator) bool {
 	return true
 }
 
-// Parse parses a schema string and returns the schema representation for it.
-func Parse(s string) (*ParsedSchema, error) {
+// parse parses a schema string and returns the schema representation for it.
+func parse(s string, namespace uint64) (*ParsedSchema, error) {
```

Comment:
Examples:
Schema 1:
```
[ns1] name: string .
[ns2] age: string .
city: string .
```

parse(schema, 0) --> All the schema fields go to namespace 0.
parse(schema, x) --> All the schema fields go to namespace x.
parse(schema, math.MaxUint64) --> name (ns1), age(ns2), city(namespace 0)

This function can be modified if we want to enforce that the schema should contain a namespace. Though not a big issue, but I think allowing schema without namespace will be helpful with data loading.
If we want the user to give either consistent schema (all fields should have namespace or none should), then we can make this code section simpler.